src/tools/fuse/CMakeLists.txt

```diff
@@ -35,10 +35,20 @@ if (NOT EXIT_CODE EQUAL 0)
 	return ()
 endif ()
 
+# Create a temporary directory to build the python package
+execute_process (
+	COMMAND mktemp -d
```

Comment:
Why are you creating a temporary directory in /tmp? I think it would be better if we use some directory in CMAKE_BINARY_DIR probably simply CMAKE_CURRENT_BINARY_DIR

Reply:
It chose `/tmp` since it was suggested in https://github.com/ElektraInitiative/libelektra/issues/4048 to not use the build directory.

Reply:
I read "should be in the build directory." and agree that the build directory is the best place :+1:

Reply:
Or are these files written during `make install`? Then the tmp directory is probably not so bad if writing these files during installation is really unavoidable. Better would be to write everything during compilation and only copy during installation.